checks/code_review.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2020 OpenSSF Scorecard Authors
+// Copyright 2023 OpenSSF Scorecard Authors
```

Comment:
you need to update this file to call the probe, see https://github.com/ossf/scorecard/blob/main/checks/security_policy.go#L47-L59

That will require updating the check/evaluation/core_review.go as well

Reply:
See https://github.com/ossf/scorecard/pull/3230 as an example PR

Reply:
I asked about it earlier but did not get a clear answer, thought these probes would only be experimental for now. Having said that, I am more than willing to update the evaluation! I need some opinions on how to score the results of the probes:
`codeApproved
codeReviewed
codeReviewTwoReviewers`?
The obvious results being:
- if all return OutcomePositive: CreateMaxScoreResult
- if all return OutcomeNegative: CreateMinScoreResult.

What about if codeApproved and codeReviewed pass, but codeReviewTwoReviewers fails, how much should they weigh when calculating the final score? How about OutcomeNotAvailable? Thankful for any input on this.

Reply:
Is this required, or can we keep both the probes and the scoring changes separate and updated in parallel?